src/connectors/youtube.js

```diff
@@ -229,9 +239,12 @@ function getTrackInfoFromDescription() {
 }
 
 function getTrackInfoFromChapters() {
-	return Util.splitArtistTrack(
-		Util.getTextFromSelectors(chapterNameSelector)
-	);
+	const chapterName = Util.getTextFromSelectors(chapterNameSelector);
+	const artistTrack = Util.splitArtistTrack(chapterName);
+	if (!artistTrack.artist && !artistTrack.track) {
```

Comment:
Could you use `isArtistTrackEmpty` from `Util` module here?

Reply:
> Could you use `isArtistTrackEmpty` from `Util` module here?

I think it is sufficient only to check whether the track name is present, so I implemented it this way instead.  Is that fine?